android/java/org/chromium/chrome/browser/crypto_wallet/fragments/onboarding_fragments/SetupWalletFragment.java

```diff
@@ -83,16 +83,16 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     // is done.
     private void checkOnBraveActivity(boolean setupAction, boolean restoreAction) {
         try {
-            BraveActivity activity = BraveActivity.getBraveActivity();
+            BraveActivity.getBraveActivity();
+        } catch (ActivityNotFoundException e) {
+            Log.e(TAG, "checkOnBraveActivity " + e);
```

Comment:
I think this won't do anything. The catch will never be executed because the return value of `BraveActivity.getBraveActivity()` is never used so doesn't matter `getBraveActivity()` returns null or not, nothing will happen. The catch block will never be executed hence it won't do anything in case of the activity is null.

Reply:
yes, maybe we just can remove `BraveActivity.getBraveActivity();` code here?

Reply:
the call `BraveActivity.getBraveActivity();` will throw an exception if the Activity is not alive and we will do the same as it used to be when the function returned `null`

Reply:
Didn't notice that, and good to go.